src/games/kalambury/board/GameBoard.js

```diff
@@ -1,10 +1,24 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
 import { Segment, Button, Header } from "semantic-ui-react";
-import { timerFormat } from "../utils/time";
+import Confetti from 'react-dom-confetti';
 import { withTranslation } from "react-i18next";
+import { timerFormat } from "../utils/time";
 import DrawingBoard from "./DrawingBoard";
 import GuessingBoard from "./GuessingBoard";
 
+const config = {
+  angle: 90,
+  spread: "54",
+  startVelocity: "56",
+  elementCount: "55",
+  dragFriction: "0.07",
+  duration: "2390",
+  stagger: 0,
+  width: "10px",
+  height: "43px",
+  colors: ["#a864fd", "#29cdff", "#78ff44", "#ff718d", "#fdff6a"]
```

Comment:
We could use out toolbar colors here. It may be a good opportunity to finally extract them to config